### ISSUE TITLE
it'd be nice if u add a certain column

### ISSUE THREAD

**Issue by mtnjustme** [n1ghty/pkg_tools#2]
package found path, it'd be nice to have that (like pkg_path or something) in an excel column.
ty in advance, and ty for your awesome work.

**Comment by plato79** [comment]
Yes, that would be really helpful.. While now we know what we have, we don't know where we have if there are several subdirectories, or some filenames are not as explanatory like others.. But I don't think n1ghty is looking here.. So I'll check to see if I could add..

**Comment by plato79** [comment]
Just solved that with https://github.com/n1ghty/pkg_tools/pull/7, you could grab it from my repo, or wait for the pull request.
